src/Traits/Elm.php

```diff
@@ -149,4 +149,14 @@ public function byXPath($value)
     {
         return $this->TestCaseElm()->by('xpath', $value);
     }
+
+    /**
+     * @param $value
```

Comment:
@anooptajawal Can you please put the reference in docblock

Reply:
@zeeshanu Added the reference link